### DISCUSSION TITLE
Trial license

### DISCUSSION THREAD

**Discussion by ADR-007** [viewflow/viewflow#385] [General]
Hello,

I'm looking for the best way to implement a project for my client. But I don't want to buy the library, just to make a 1-day PoC.
Is there a way to receive the library for temporal use? Maybe you have a Trial or non-commercial license for the "pro" version?

Thank you!

**Comment by kmmbvnr** [comment]
Hi Adrian,

Thank you for your interest in our Viewflow library. I understand the importance of evaluating the right tools for your project before making a purchase. The best way to start would be to create a PoC using our APGL version. You can leverage this version to familiarize yourself with the basic features of Viewflow. Once you have your PoC ready, please send your models.py and flows.py files to my email address at [mikhail@viewflow.io](mailto:mikhail@viewflow.io). Along with these files, kindly include a detailed list of the pro features you are interested in demonstrating to your client.

I would be more than happy to assist you further, and we can discuss potential trial possibilities for the pro version based on your specific requirements.

Mikhail
